### PR TITLE
2380 tof widget

### LINKED ISSUE #2380: Image Window: Place Time of Flight (ToF) Properties into a Widget
This issue is in preparation for Bragg Edge development. We will likely be altering the Spectrum Viewer GUI significantly.
Due to this volatility, we should place existing components into Widgets so that it is easy to re-use where applicable throughout other sections of the Spectrum Viewer in the future if needed and also allow for easier relocation of existing components.

Place Time of Flight (ToF) properties into a widget:

Image

### PR DESCRIPTION
**Pull request by JackEAllen**
### Issue

Closes #2380 

### Description

 Rename Spectrum Viewer `tofPropertiesGroupBox` to `experimentSetupGroupBox` and place into a custom widget (`ExperimentSetupFormWidget`)

### Testing 

*Describe the tests that were used to verify your changes*.
* Unit test
* manual testing:
    * Verified that experimentSetupGroupBox is correctly disabled when no ToF data available
    * Verify that  ExperimentSetupFormWidget correctly handles value changed signal for any widgets added to `connect_value_changed()`

### Acceptance Criteria 

* Unit tests pass
* experimentSetupGroupBox is correctly disabled when no ToF data available
* ExperimentSetupFormWidget correctly handles value changed signal and update GUI

### Documentation

*How have you changed the documentation to reflect your changes? All changes should be noted in the appropriate file in docs/release_notes*

`docs/release_notes/next/dev-2380-ExperimentSetupFormWidget`

### CODE REVIEW

File: mantidimaging/gui/ui/spectrum_viewer.ui [hunk 1]
```diff
@@ -440,54 +440,8 @@
...
-                    <widget class="QDoubleSpinBox" name="timeDelaySpinBox">
-                    <property name="suffix">
-                    <string/>
-                    </property>
-                    </widget>
-                </item>
-            </layout>
-        </widget>
-    </item>
+            <widget class="QGroupBox" name="experimentSetupGroupBox"></widget>
```
Comment (samtygier-stfc) [inline mantidimaging/gui/ui/spectrum_viewer.ui:443]: Does changing this to
`<widget class="QWidget" name="experimentSetupGroupBox"></widget>`
fix the screen shot change?

Comment (JackEAllen) [reply mantidimaging/gui/ui/spectrum_viewer.ui:443]: Yes. using QWidget will likely resolve the screenshot difference as QWidget does not have a border set by the property setFlat which is by default set to true in QGroupBox

**Review by samtygier-stfc — APPROVED** [review]
Tested locally, code looks good. Happy to accept a small GUI change.

**Comment by JackEAllen** [comment]
This PR changes the Spectrum Viewer window slightly, therefore I have created a new baseline for the screenshot tests

**Comment by JackEAllen** [comment]
> Tested locally, code looks good. Happy to accept a small GUI change.

I have placed the spinboxes and labels in a QGridLayout so the space between components is justified equally which makes the visual difference compared to the previous implementation smaller. Placing in a QGridLayout should hopefully make it easier to add more components in the future.

**Pull request merged**